stardust/env_template

```diff
@@ -1,9 +1,13 @@
 # This is an example configuration file.
+# Keep this file as an example and make a working file with the following command:
 #
-# Uncomment the lines to fit your needs and rename the file to ".env" at the end
-# with the following command:
+#   cp env_template .env
 #
-#   mv env_template .env 
+# You can edit the .env file with a command like: 
+#
+#   nano .env 
+#
+# The .env file is your personal configuration and is used by Docker.
```

Comment:
We think we should keep the uncomment message, wdyt?
```suggestion
# The .env file is your personal configuration and is used by Docker.
# Uncomment and edit the lines to fit your needs
```

Reply:
Btw if we apply this, you don't need to create another PR in the wiki, I will just import this file into the wiki